service-statistics-common/src/main/java/fi/nls/oskari/control/statistics/plugins/db/StatisticalDatasource.java

```diff
@@ -18,6 +19,7 @@
     private String config;
     private String plugin;
     private List<DatasourceLayer> layers = new ArrayList<>();
+    private long maxUpdateDuration = TimeUnit.HOURS.toMillis(6);
     private long updateInterval = 1000 * 60 * 60 * 24;
```

Comment:
Change this to TimeUnit as well so it doesn't feel out of place with maxUpdateDuration

Reply:
Done